src/refract/RenderJSONVisitor.cc

```diff
@@ -102,10 +105,38 @@ namespace refract
         if (!a || a->value.empty()) {
             return NULL;
         }
+        
+        std::cerr << "samples" << std::endl;
 
         return TypeQueryVisitor::as<T>(*(a->value.begin()));
     }
 
+    template<typename T>
+    const T* getNullable(const T& e) {
+        IElement::MemberElementCollection::const_iterator ta = e.attributes.find("typeAttributes");
+        
+        if (ta == e.attributes.end()) {
+            return NULL;
+        }
+        
```

Comment:
Still trying to work through the serialized attributes to find "nullable" when it exists.